json_database/crypto.py

```
@@ -0,0 +1,112 @@
+        return decrypt(key, ciphertext, tag, nonce)
+    except:
+        raise DecryptionKeyError
+
+
+def encrypt_bin(key, data):
+    if len(key) > 16:
+        key = key[0:16]
+    try:
+        data = compress_payload(data)
+        ciphertext, tag, nonce = encrypt(key, data)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)